src/js/view/utils/views-manager.js

```diff
@@ -103,7 +102,6 @@ if (isAndroidChrome && hasOrientation) {
 window.addEventListener('beforeunload', () => {
     document.removeEventListener('visibilitychange', onVisibilityChange);
     document.removeEventListener('webkitvisibilitychange', onVisibilityChange);
-    window.removeEventListener('resize', scheduleResponsiveRedraw);
     window.removeEventListener('orientationchange', scheduleResponsiveRedraw);
```

Comment:
left this in - but debating it. not sure if it's redundant like `resize` with the changes here. 

also are there concerns with moving from the `scheduleResponsiveRedraw` function here to `_responsiveListener` (which also schedules in a way)

Reply:
`scheduleResponsiveRedraw` notifies all players that they need to check if they've been resized. If each player has it's own `ResizeListener` then `scheduleResponsiveRedraw` and the "orientationchange" listener are no longer needed.

Reply:
Will remove!